src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -856,11 +856,11 @@ function MOI.get(model::Optimizer, ::MOI.TerminationStatus)
     end
 end
 
-# TODO(odow): handle solutions that may exist when limit reached.
 function MOI.get(model::Optimizer, attr::MOI.PrimalStatus)
     if attr.N != 1
         return MOI.NO_SOLUTION
-    elseif MOI.get(model, MOI.TerminationStatus()) == MOI.OPTIMAL
+    end
+    if MOI.get(model, MOI.ResultCount()) > 0
         return MOI.FEASIBLE_POINT
```

Comment:
There might be a solution but can Cbc certify that it is feasible ? Maybe we should have a fourth case that returns `MOI.UNKNOWN_RESULT_STATUS`

Reply:
I had problems with time limits and problems not reaching desired gap. 
Cbc.jl returned NO_SOLUTION allthough integer solutions were found, proposed change would fix that. 
Can you describe situation in which bestSolution() is not NULL and solution is not feasible?

Reply:
I think the question is if the Cbc API makes it clear that the solution returned in the situation is feasible. I would guess this is true.